tests/test_diffraction.py

```diff
@@ -0,0 +1,51 @@
+import freud
+import matplotlib
+import unittest
+import numpy.testing as npt
+matplotlib.use('agg')
+
+
+class TestDiffractionPattern(unittest.TestCase):
```

Comment:
We should add docstrings to the unit tests. @vyasr has mentioned to me that at some point he plans on having the CI style-check use more flake8 plugins, so to get out ahead of it complaining about the lack of docstrings in unit tests, we should plan on adding them now so we don't have to fix them later.

Reply:
@tommy-waltmann I'm not sure I know what you're wanting to write here. Feel free to make suggestions, but I haven't written docstrings for tests in any package before.